apps/shark_studio/api/llm.py

```diff
@@ -130,13 +135,18 @@ def __init__(
                 print(
                     f"External weight file {self.external_weight_file} does not exist. Generating..."
                 )
-                gen_external_params(
-                    hf_model_name=self.hf_model_name,
-                    quantization=self.quantization,
-                    weight_path=self.external_weight_file,
-                    hf_auth_token=hf_auth_token,
-                    precision=self.precision,
+                # gen_external_params(
+                #    hf_model_name=self.hf_model_name,
+                #    quantization=self.quantization,
+                #    weight_path=self.external_weight_file,
+                #    hf_auth_token=hf_auth_token,
+                #    precision=self.precision,
+                # )
+                cache_dir = os.path.join(".", str(self.hf_model_name).replace("/", "_"))
+                huggingface_hub.snapshot_download(
+                    repo_id=self.hf_model_name, cache_dir=cache_dir
                 )
+                # TODO: Convert to gguf, delete cache
```

Comment:
The way that sharktank recommends for generating the .gguf file is to use a CLI tool from llama.cpp. Is that still the best way to extract that, or do we have a way to do it using sharktank?